assets/docs/rishit-dagli/mirnet-tfjs/1.md

```diff
@@ -0,0 +1,27 @@
+# Placeholder rishit-dagli/mirnet-tfjs/1
+MIRNet model to enhance a low-light images.
+
+<!-- module-type: image-super-resolution -->
+<!-- network-architecture: other -->
+<!-- dataset: lol -->
```

Comment:
https://raw.githubusercontent.com/tensorflow/tfhub.dev/master/assets/docs/sayakpaul/mirnet-fixed/1.md uses `LOL`, so maybe for consistency reasons capitalize the dataset name.

Reply:
Thanks for pointing this out, this has now been fixed